snowfall-compose/src/commonMain/kotlin/io/github/skeptick/snowfall/compose/internal/SnowflakeState.kt

```diff
@@ -28,13 +31,14 @@ internal fun SnowflakeState(
     canvasSize: Size,
     pathSize: Float,
     size: ClosedRange<Float>,
-    speed: ClosedRange<Float>
+    speed: ClosedRange<Float>,
 ): SnowflakeState {
     val scaleRatio = Random.nextFloat()
     val speedRatio = Random.nextFloat()
     return SnowflakeState(
         x = Random.nextFloat() * canvasSize.width,
         y = Random.nextFloat() * canvasSize.height * -1f,
+        alphaOffset = Random.nextFloat() * 0.5f + 0.5f,
```

Comment:
Magic numbers should be extracted into constants. It would be better to implement this similarly to how the angle is calculated, so the value is taken from a range between 0.5 and 1.